### PR TITLE
chore: add clock.next()

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: docs/src/api/class-clock.md [hunk 1]
```diff
@@ -42,6 +43,14 @@ Tells `@sinonjs/fake-timers` to increment mocked time automatically based on the
 Relevant only when using with [`option: shouldAdvanceTime`]. Increment mocked time by advanceTimeDelta ms every advanceTimeDelta ms change
 in the real system time (default: 20).
 
+
+## async method: Clock.next
+* since: v1.45
+- returns: <[int]> Fake milliseconds since the unix epoch.
+
+Advances the clock to the the moment of the first scheduled timer, firing it.
```
Comment (dgozman) [inline docs/src/api/class-clock.md:51]: What happens if there are no timers scheduled?

File: tests/page/page-clock.spec.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+import { describe } from 'node:test';
```
Comment (dgozman) [inline tests/page/page-clock.spec.ts:17]: Please fix 😃

File: tests/page/page-clock.spec.ts [hunk 3]
```diff
@@ -612,3 +613,113 @@ it.describe('shouldAdvanceTime', () => {
...
+      }, 0);
+    });
+
+    await page.clock.next();
+    expect(calls).toEqual([{ params: [1] }]);
+    await page.clock.next();
+    expect(calls).toEqual([{ params: [1] }, { params: [2] }]);
+  });
+
+  it('throws exception thrown by timer', async ({ page, calls }) => {
```
Comment (dgozman) [inline tests/page/page-clock.spec.ts:715]: This behavior deserves being documented!

**Review by dgozman — APPROVED** [review]

File: tests/page/page-clock.spec.ts (hunk 3 above)
Comment (pavelfeldman) [reply tests/page/page-clock.spec.ts:714]: Will follow up

File: docs/src/api/class-clock.md (hunk 1 above)
Comment (pavelfeldman) [reply docs/src/api/class-clock.md:51]: I'll need to follow up to respond.

**Pull request merged**